docs/guide/advanced/shader/shader_compute.md

```diff
@@ -154,37 +154,37 @@ this.mRTFrame = new RTFrame([
     descript
 ]);
 
-// 将该纹理关联到ComputeShader
+// Associate the texture with the ComputeShader
 this.mGaussianBlurShader.setStorageTexture(`resultTex`, this.mBlurResultTexture);
 ```
 
-到这里，`ComputeShader`的初始化，相关 `GPU Buffer` 和 `Texture` 的创建与关联都已完成，接下来是执行 `ComputeShader`，在执行之前，我们还需要根据需求设置好派发调度时工作组数量，也就是参数 `workerSizeX`、
-`workerSizeY`、`workerSizeZ`：
+At this point, the initialization of the`ComputeShader`, creation and association of relevant `GPU Buffer` and `Texture` have been completed. Next, we need to execute the `ComputeShader`. Before executing, we need to set the number of workgroups according to the requirements, which are the parameters `workerSizeX`,
```

Comment:
and the creation and the association of ...
or
the initialization of xxx has been finished, and the creation and xxxx have been completed